src/event_timer.py

```diff
@@ -35,6 +35,8 @@ class EventTimer:
         unit_name (str): Name of the juju unit to emit events to.
     """
 
+    _juju_run_path = Path("/usr/bin/juju-run")
+    _juju_exec_path = Path("/usr/bin/juju-exec")
```

Comment:
May I ask where these are used? I see the additions but I don't see any references to the internal variables (perhaps github rendering issue?)

Reply:
Removed